### PR TITLE
Consistent Changes

### PR DESCRIPTION
**Pull request by auguwu**
# Fixes:
- Changing -> Changed
- Use Template Literals
- Instead of using `bool == true`, use `msg.author.bot` (since bot is a boolean)

### CODE REVIEW

**Comment by davidcralph** [comment]
+1

File: bot.js [hunk 1]
```diff
@@ -3,17 +3,17 @@ const client = new Discord.Client();
...
 
 client.on('message', msg => {
-  if (msg.author.bot == true) return;
+  if (msg.author.bot)
+      return;
   const randPasta = pasta[Math.floor(Math.random()*pasta.length)];
   // if (pasta.indexOf(msg.member.displayName) == -1) {
     msg.member.setNickname(randPasta)
-      .then(console.log("Changing " + msg.author.username + "'s nickname"))
+      .then(console.log(`Changed ${msg.author.username}'s nickname to ${randPasta}!`))
```
Comment (oliyy) [inline bot.js]: Still says changed even if an error occurs and it didn't actually change

File: bot.js [hunk 2]
```diff
@@ -3,17 +3,17 @@ const client = new Discord.Client();
...
 const pasta = ["Spaghetti", "Macaroni", "Ravioli", "Cannelloni", "Linguine", "Lasagne", "Rigatoni", "Tortellini", "Penne", "Fettuccine", "Pappardelle", "Orzi", "Bucatini", "Fusilli", "Vermicelli", "Orecchiette", "Fregula", "Taglierini", "Farfalle", "Manicotti", "Cavatelli", "Paccheri", "Capellini", "Agnolotti", "Strozzapreti", "Cavatappi", "Ziti", "Trofie", "Garganelli", "Conchiglie", "Casarecce", "Rotini", "Pizzoccheri", "Ditalini", "Ptitim", "Gemelli", "Scialatielli", "Cappelletti", "Capelli d’angelo", "Bigoli", "Calamarata", "Mezzelune", "Bavette", "MeMes"]
 
-
 client.on('ready', () => {
   console.log(`Logged in as ${client.user.tag}!`);
 });
 
 client.on('message', msg => {
-  if (msg.author.bot == true) return;
+  if (msg.author.bot)
```
Comment (oliyy) [inline bot.js:11]: Yes, thanks I was thinking about this when I was lying in bed.

**Review by oliyy — CHANGES_REQUESTED** [review]

File: bot.js (hunk 1 above)
Comment (auguwu) [reply bot.js]: Ok! Will change.

**Comment by auguwu** [comment]
@Oliyy request a new review :)

File: bot.js (hunk 2 above)
Comment (auguwu) [reply bot.js:11]: No problem 👌

**Review by oliyy — APPROVED** [review]

**Pull request merged**